src/bin/rustc_josh_sync.rs

```diff
@@ -109,10 +109,19 @@ fn main() -> anyhow::Result<()> {
                 .context("cannot perform push")?;
 
             // Open PR with `subtree update` title to silence the `no-merges` triagebot check
+            let merge_msg = format!(
+                r#"Subtree update of https://github.com/{}.
+
+Created using https://github.com/rust-lang/josh-sync.
+
+r? @ghost"#,
```

Comment:
Since we have the subrepo sha, mind adding a link to that commit in the message? I've found that helpful to easily find the latest sync point, e.g. https://github.com/rust-lang/rust/pull/143405.

Reply:
Indeed 😆 I meant whatever the current local HEAD in the subtree repo is for a `https://github.com/{subtree_org}/{subtree_repo}/commit/{sha}` url. So you can click it from the sync-to-monorepo PR and see what exactly you're syncing to

I just used the HEAD from the check against Josh https://github.com/rust-lang/compiler-builtins/blob/ed17b95715ddce362fdea2c787e6efb28824f29c/crates/josh-sync/src/sync.rs#L230-L248

Reply:
![image](https://github.com/user-attachments/assets/68ae3221-a573-46d3-b989-ef84438969a5)

Changed it like this.